rfc/README.md

```diff
@@ -86,3 +86,4 @@ The list of all RFCs can be found here.
 | 48 | [LogCompaction for MOR tables](./rfc-48/rfc-48.md) | `UNDER REVIEW` | 
 | 49 | [Support sync with DataHub](./rfc-49/rfc-49.md)    | `ONGOING` |
 | 50 | [Improve Timeline Server](./rfc-50/rfc-50.md) | `UNDER REVIEW` | 
+| 52 | [Support Lucene Based Record Level Index](./rfc-52/rfc-52.md) | `UNDER REVIEW` |
```

Comment:
Could you create an individual PR to pick the RFC number, to follow the [Hudi RFC process](https://hudi.apache.org/contribute/rfc-process)?

Reply:
PR for claim RFC 52 has been merged, I will rebase master later. Thanks.